projects/kit/utils/mask/create-time-mask.ts

```diff
@@ -3,19 +3,34 @@ import {TUI_DIGIT_REGEXP, TuiTextMaskList} from '@taiga-ui/core';
 import {MAX_TIME_VALUES} from '@taiga-ui/kit/constants';
 import {TuiTimeFormatParts} from '@taiga-ui/kit/types';
 
+function tuiCreateTimePartMask(
+    maxPartValue: number | undefined,
+    prefix?: string,
+): RegExp[] {
+    const partLength =
+        typeof maxPartValue === 'number' ? maxPartValue.toString().length : 0;
+
+    if (!partLength) {
+        return [];
+    }
+
+    const regExp = Array(partLength).fill(TUI_DIGIT_REGEXP);
```

Comment:
please, use `Array.from({ length: partLength })` instead of `Array(partLength)` 

for example - https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-new-array.md